deploy/a8s/manifests/postgresql-images.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  backupAgentImage: public.ecr.aws/w5n9a2g2/a9s-ds-for-k8s/dev/backup-agent:81450cbb7f94fa020441361d8c64ew89499acd13
+  backupAgentImage: public.ecr.aws/w5n9a2g2/a9s-ds-for-k8s/dev/backup-agent:781ca7b6e9bb4af66aeedb5046b50dc1c2b6c638
```

Comment:
How is this change related to the OOMKill?

Reply:
Should I just drop the commit?

Reply:
Do you have a suggestion?

Reply:
let's drop it. @abdulhaseeb3 can you look into why this change was made? you updated the images two weeks ago, and this would revert some of those changes